app/views/manage_submissions/_phases_table.html.erb

```diff
@@ -15,7 +15,7 @@
               <%= challenge_phase_title(challenge, phase) %>
               </th>
             <td data-label="Number of Submissions">
-              <%= phase.submissions.length %>
+              <%= phase.submissions_count %>
```

Comment:
is this supposed to count the total number of submissions or just the submissions that have been assigned to an evaluator? I think it's the former..